reference/sql_workarounds.rst

```diff
@@ -0,0 +1,141 @@
+.. _workarounds:
+
+SQL Workarounds
+===============
+
+Citus supports most, but not all, SQL statements directly. Its SQL support continues to improve. Also many of the unsupported features have workarounds; below are a number of the most useful.
+
+Subqueries
+----------
+
+In analytics it is common to model information using a "star schema." This is a collection of dimensional tables (e.g. observations, statistics) referencing a table of facts (e.g. people, pages or events). Furthermore, a common form of query examines dimensions for a subset of the facts. Examples of these kind of queries:
```

Comment:
Im not sure whether it is a good idea to give the fact-dimension table example. In customer use-cases we have not seen this. We have seen this in more complex cases, such as on the same table or between 2 large tables which are not colocated on the column which has a filter on subquery, etc.  In the fact-dimension table example, the most simplest workaround is by a broadcast join because dimension tables are most of the times small.

One idea which comes to my mind to rewrite this description, is by writing the actual queries for each customer use-cases and mention that these are some examples where subqueries in where clause kicks in. 

And in the actual work arounds, take one of the examples and give workarounds for that example.
@samay-sharma @sumedhpathak : Could you give your inputs on this?